src/applications/personalization/profile360/components/ContactInformation.jsx

```diff
@@ -4,29 +4,29 @@ import React from 'react';
 import DowntimeNotification, { externalServices } from '../../../../platform/monitoring/DowntimeNotification';
 import recordEvent from '../../../../platform/monitoring/record-event';
 import accountManifest from '../../account/manifest.json';
-import { FIELD_NAMES, TRANSACTION_CATEGORY_TYPES } from '../constants/vet360';
+import { TRANSACTION_CATEGORY_TYPES } from '../constants/vet360';
 import Vet360PendingTransactionCategory from '../containers/Vet360PendingTransactionCategory';
-import PhoneSection from './PhoneSection';
-import AddressSection from './AddressSection';
-import EmailSection from './EmailSection';
 import LoadFail from './LoadFail';
 import { handleDowntimeForSection } from './DowntimeBanner';
 import MissingVet360IDError from './MissingVet360IDError';
 import ContactInformationExplanation from './ContactInformationExplanation';
 
-export default class ContactInformation extends React.Component {
+import MailingAddress from '../vet360/components/MailingAddress';
+import ResidentialAddress from '../vet360/components/ResidentialAddress';
+import HomePhone from '../vet360/components/HomePhone';
+import MobilePhone from '../vet360/components/MobilePhone';
+import WorkPhone from '../vet360/components/WorkPhone';
+import FaxNumber from '../vet360/components/FaxNumber';
+import Email from '../vet360/components/Email';
 
-  componentDidMount() {
-    this.props.fetchAddressConstants();
```

Comment:
Noting that is included because an endpoint does exist for us to dynamically pull state and country names. Switching to a hardcoded file because we do not expect the states/countries to have changes in the near term.